src/components/CodeBlock/index.js

```diff
@@ -0,0 +1,176 @@
+import * as React from "react";
+import Highlight, { defaultProps } from "prism-react-renderer";
+import Confetti from "react-dom-confetti";
```

Comment:
lol we didn't need the confetti but nbd. does it look good

Reply:
I want the confetti lmao. It looks good

Reply:
Though it's not really necessary..... but it's confetti!

Reply:
confetti!!!